bgc_data/bgc_zooplankton_abundance_genus_data.sql

```diff
@@ -11,7 +11,8 @@ WITH grouped AS (
            substring(taxon_name, '^\w+') != substring(parent_name, '^\w+') AS genus_changed,
            sum(r.zoop_abundance_m3) AS zoop_abundance_m3
     FROM bgc_zoop_raw r LEFT JOIN bgc_zoop_changelog c USING (taxon_name)
-    WHERE r.genus IS NOT NULL
+    WHERE r.genus IS NOT NULL AND 
+          r.genus NOT LIKE '%cf.%'
```

Comment:
@clairedavies Just checking, is this correct? I don't see any such filtering being done in your [R code](https://github.com/PlanktonTeam/planktonr/blob/80b245144ce3b101a6d3a402294a66f48eb5c2ea/R/utils_nrs.R#L741) for the Genus level product (and same for phyto abundance and biovolume).

Reply:
Claire has confired by email that this is correct, so we can merge it!